benches/bn256_field.rs

```diff
@@ -43,6 +42,12 @@ pub fn bench_bn256_field(c: &mut Criterion) {
     group.bench_function("bn256_fq_invert", |bencher| {
         bencher.iter(|| black_box(&a).invert())
     });
+    group.bench_function("bn256_fq_legendre", |bencher| {
```

Comment:
I assume Legendre is the old implementation?

Reply:
Yes.